### DISCUSSION TITLE
How to rename a file in a repository using the website

### DISCUSSION THREAD

**Discussion by WillPittenger** [community/community#68398] [Repositories] [answered]
### Select Topic Area

Question

### Body

I dropped a folder's worth of images onto my repository in my browser unaware that one of them had the wrong file name extension.  Now I want to rename it to fix the issue.  Can I do that from GitHub?  Or do I need to delete it and add it back with the correct extension?

**Comment by radar-blip** [comment, marked as answer]
Not sure if you can do it within GitHub.
Its probably best (and just as easy) to delete it, rename it, and add it back again.

**Reply by WillPittenger** [reply]
OK.   Thanks.

**Reply by radar-blip** [reply]
Anytime.
